MadeByMunstersLLC/fuel
# adds basic faq support to admin interface that includes faq categories.

**Changes:**
Adds basic support for faq's to the fuel gem. This includes creating faq categories to group faq's by.

**Testing:**
1. Pull down the branch.
2. cd into `/spec/dummy`
3. run `rails s`
4. login to admin interface at `http://127.0.0.1:3000/blog/admin`
5. Test to make sure that you can add Faq Categories and add Faq's to that category.
6. Route to `http://127.0.0.1:3000/blog/faqs.json` and `http://127.0.0.1:3000/blog/faq_categories.json` to verify the response from the server.
